tests/modules/nf-core/gawk/nextflow.config

```diff
@@ -0,0 +1,10 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: GAWK {
+        ext.prefix = {"${meta.id}.bed"}
+        ext.args = '\'BEGIN {FS="\t"}; {print \$1 FS "0" FS \$2}\''
```

Comment:
Should this just be a val input instead?

Reply:
That's also a possibility, but isn't really following the `nf-core` guidelines. I do agree that this is a gray area though

Reply:
Is it that different then `bedtools_intersect`? Maybe 
```nextflow
    tuple val(meta), path(input), val(pattern)
```

Reply:
This has now been fixed with the program file option. So both options are possible now